backend/src/main/java/ubb/courses/backend/repositories/EnrollmentRepository.java

```diff
@@ -10,4 +10,6 @@
 public interface EnrollmentRepository extends JpaRepository<Enrollment, Integer> {
 
     List<Enrollment> deleteAllByCourseId(Integer id);
+
+    Enrollment findByCourseIdInAndUserIdIn(int course_id, int user_id);
```

Comment:
`findByCourseIdInAndUserId` should do it

Reply:
should be good now